src/pages/[productSlug]/tutorials/index.tsx

```diff
@@ -13,21 +14,9 @@ import { cachedGetProductData } from 'lib/get-product-data'
  * i.e. /vault/tutorials
  */
 function generateProductTutorialHomePaths() {
-	const paths = []
-
-	/**
-	 * @TODO ignoring `hcp` product slug for now until we know whether or not
-	 * we're using "hcp" or "cloud".
-	 * https://app.asana.com/0/1202097197789424/1202618936981037/f
-	 */
-	__config.dev_dot.beta_product_slugs.forEach((productSlug: ProductSlug) => {
-		if (productSlug !== 'hcp') {
-			paths.push({
-				params: { productSlug },
-			})
-		}
-	})
-
+	const paths = __config.dev_dot.beta_product_slugs.map(
+		(productSlug: ProductSlug) => ({ params: { productSlug } })
+	)
```

Comment:
`hcp` is now marked as out of beta in preview, so we no longer want to manually ignore it.